samples/android/app/build.gradle

```diff
@@ -36,8 +36,7 @@ dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    implementation "com.android.support:appcompat-v7:27.0.0"
-    implementation "com.android.support:customtabs:27.0.0"
+    implementation "androidx.appcompat:appcompat:1.0.0"
```

Comment:
This change was needed to be compatible with `android-components`, which is now being brought into the sample app as a dependency of glean-core.